tests/test_headers.py

```diff
@@ -692,6 +704,28 @@ def test_content_length_options(self, client):
         content_length = '0'
         assert result.headers['Content-Length'] == content_length
 
+    def test_disabled_cors_should_not_add_any_extra_headers(self, client):
+        client.app.add_route('/', CORSHeaderResource())
+        result = client.simulate_get()
+        assert 'Access-Control-Allow-Origin'.lower() not in dict(
+            result.headers.lower_items()).keys()
+
+    def test_enabled_cors_should_add_extra_headers_on_response(self, cors_client):
+        cors_client.app.add_route('/', CORSHeaderResource())
+        result = cors_client.simulate_get()
+        assert 'Access-Control-Allow-Origin'.lower() in dict(
+            result.headers.lower_items()).keys()
+
+    def test_enabled_cors_should_accept_all_origins_requests(self, cors_client):
+        cors_client.app.add_route('/', CORSHeaderResource())
+        result = cors_client.simulate_get()
+        assert result.headers['Access-Control-Allow-Origin'] == '*'
+
+    def test_enabled_cors_should_control_expiration_through_headers(self, cors_client):
+        cors_client.app.add_route('/', CORSHeaderResource())
+        result = cors_client.simulate_options(headers=(('Access-Control-Request-Method', '*'), ))
+        assert result.headers['Access-Control-Max-Age'] == '86400'  # 24 hours in minutes
```

Comment:
FWIW it is 24 hours in seconds :slightly_smiling_face:

Reply:
You are right! Fixed it! thanks :)